releases/release-1.22/README.md

```diff
@@ -34,52 +34,61 @@ description: |
 
 ## TL;DR
 
-TBD
+The 1.22 release cycle is proposed as follows:
+
+- **Monday, April 26th**: Week 1 - Release cycle begins
+- **Thursday, May 13th**: Week 3 - [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **Thursday, July 8th**: Week 11 - [Code Freeze](../release_phases.md#code-freeze)
+- **Thursday, July 15th**: Week 12 - [Test Freeze](../release_phases.md#test-freeze)
+- **Wednesday, July 27th**: Week 14 - Docs must be completed and reviewed
+- **Wednesday, August 4th**: Week 15 - Kubernetes v1.22.0 released
+- **TBD**: Week 16 - Release Retrospective
 
 ## Timeline
 
 | **What** | **Who** | **When** | **WEEK** | **CI SIGNAL** |
 |---|---|-------|---|---|
 | Start of Release Cycle | Lead | Mon April 26 | week 1 | [master-blocking] |
-| Start Enhancements Tracking | Enhancements Lead | TBD | TBD | |
+| Start Enhancements Tracking | Enhancements Lead | Tue April 26 | week 1 | |
```

Comment:
```suggestion
| Start Enhancements Tracking | Enhancements Lead | Tue April 27 | week 1 | |
```

Reply:
It is intended to start on the 26th. Will change the day to reflect the same